packages/uniforms-bridge-json-schema/src/JSONSchemaBridge.ts

```diff
@@ -271,6 +283,14 @@ export default class JSONSchemaBridge extends Bridge {
       }
     }
 
+    Object.keys(ready).forEach(key => {
+      if (isKey(key)) {
```

Comment:
If you'd add `: Record<string, string>` to `propMapper`, then this function could be replaced with `key in propMapper`.